README.md

```diff
@@ -308,6 +308,107 @@ Descope SDK is automatically refreshes the session token when it is about to exp
 If the Descope project settings are configured to manage tokens in cookies.
 you must also configure a custom domain, and set it as the `baseUrl` prop in the `AuthProvider` component. See the above [`AuthProvider` usage](https://github.com/descope/react-sdk#wrap-your-app-with-auth-provider) for usage example.
 
+### Widgets
+
+Widgets are components that allow you to expose management features for tenant-based implementation. In certain scenarios, your customers may require the capability to perform managerial actions independently, alleviating the necessity to contact you. Widgets serve as a feature enabling you to delegate these capabilities to your customers in a modular manner.
+
+Important Note:
+
+- For the user to be able to use the widget, he needs to be assigned the `Tenant Admin` Role.
+
+#### User Management
+
+The `UserManagement` widget will let you embed a user table in your site to view and change information, exposing the following fields:
+
+`LoginId`, `Status`, `Name`, `Email`, `Phone`, `Roles`
+The widget also lets you:
+
+- Create a new user.
+- Edit an existing user.
+- Activate / disable an existing user.
+- Reset an existing user's password.
+- Reset an existing user's passkey.
+- Delete an existing user.
+
+Note:
+
+- Custom fields will also appear in the table.
+
+###### Usage
+
+```js
+import { UserManagement } from '@descope/react-sdk';
+...
+			<UserManagement
+                widgetId="user-management-widget"
+				tenant="your-tenant-id"
+			/>
+```
+
+Example:
+[Manage Users](./examples/app/ManageUsers.tsx)
+
+#### Role Management
+
+The `Role Management` widget lets you delegate the role management to your customer.
+Not only viewing the roles but also adding permissions, renaming, and changing the description of an existing role.
+The widget exposes this information about the roles:
+
+`Name`, `Description`, `Permissions`, `Editable`
+
+The widget also lets you:
+
+- Create new roles.
```

Comment:
In the user section above you use singular and here plural, use singular here as well.